examples/node8-yarn/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM outrigger/node:8-build as builder
+
+# Only perform the yarn install if package.json or the lockfile changes.
+# The insistence on asterisks for optional files prevents errors.
+# By only attempting to copy some of the custom code, Docker will not
+# reset the cache for this layer unless these specific files have changed.
+COPY ./package.json ./yarn* ./.yarn* /code/
+# Install our dependencies. Clean the cache just because.
+RUN yarn install && yarn cache clean
+# For npm, remove the above and instead use this:
+# COPY ./package.json ./pakage* ./npm* ./.npm* /code/
+# RUN npm install
+
+FROM outrigger/node:8
```

Comment:
Need to fix this to the `2-node8` version scheme... if we decide to go forward with that.